package-lock.json

```diff
@@ -1,18007 +1,8 @@
 {
   "name": "react-p5-wrapper",
   "version": "2.4.1",
-  "lockfileVersion": 2,
```

Comment:
Lock files should always be committed but not sure why the version key is gone, doesn't matter much since the package.json is all that matters, this file is just used by npm itself anyway and no dependencies changes so if npm changed this, I am sure there is a reason, right?